testdata/smoke_config.yml

```diff
@@ -53,14 +53,3 @@ keys:
 repotracker:
     numnewreporevisionstofetch: 10
     maxreporevisionstosearch: 10
-
```

Comment:
I removed the following section because it doesn't do anything and it confused me.

But maybe it is being used somewhere I am not aware of ?